deploy/crds/operator.ibm.com_v1alpha1_metaoperatorconfig_cr.yaml

```diff
@@ -4,6 +4,16 @@ metadata:
   name: common-service
 spec:
   services:
+  - name: ibm-metering-operator
+    spec:
+      Metering:
```

Comment:
can you change the first letter to lower case? Likely the kubebuilder does not allow upper case ini first letter.

Reply:
done